File: vieroShoesApp/app/src/main/java/com/cindodcindy/vieroshoes/view/MenuBaruActivity.java
package com.cindodcindy.vieroshoes.view;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.cindodcindy.vieroshoes.R;

public class MenuBaruActivity extends AppCompatActivity {

    private CardView cardView_list, cardView_order, cardView_payment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_menu_baru);

        cardView_list=findViewById(R.id.cv_go_to_list_sepatu);
        cardView_order=findViewById(R.id.cv_go_to_order);
        cardView_payment=findViewById(R.id.cv_go_to_payment);

        cardView_list.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MenuBaruActivity.this,HomeActivity.class);
                startActivity(intent);
            }
        });

        cardView_order.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent intent = new Intent(MenuBaruActivity.this,OrderActivity.class);
                startActivity(intent);


            }
        });

        cardView_payment.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MenuBaruActivity.this,PaymentListActivity.class);
                startActivity(intent);

            }
        });
    }
}

File: vieroShoesApp/app/src/main/java/com/cindodcindy/vieroshoesnew/view/MenuBaruActivity.kt
package com.cindodcindy.vieroshoesnew.view

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.cindodcindy.vieroshoesnew.R
import com.cindodcindy.vieroshoesnew.view.activity.AboutActivity
import com.cindodcindy.vieroshoesnew.view.activity.ProductActivity
import com.cindodcindy.vieroshoesnew.view.activity.ProfileActivity
import com.orhanobut.hawk.Hawk
import kotlinx.android.synthetic.main.activity_menu_baru.*

class MenuBaruActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu_baru)

        cv_go_to_list_sepatu.setOnClickListener { v: View? ->
            val intent = Intent(this@MenuBaruActivity, ProductActivity::class.java)
            startActivity(intent)
        }
        cv_go_to_order.setOnClickListener { v: View? ->
            val intent = Intent(this@MenuBaruActivity, OrderActivity::class.java)
            startActivity(intent)
        }
        cv_go_to_payment.setOnClickListener { v: View? ->
            val intent = Intent(this@MenuBaruActivity, PaymentListActivity::class.java)
            startActivity(intent)
        }

        cardViewProfile.setOnClickListener {
            val intent = Intent(this@MenuBaruActivity, ProfileActivity::class.java)
            startActivity(intent)
        }

        cardViewAbout.setOnClickListener {
            val intent = Intent(this@MenuBaruActivity, AboutActivity::class.java)
            startActivity(intent)
        }

        cardViewLogout.setOnClickListener {
            val intent = Intent(this@MenuBaruActivity, Login::class.java)
            startActivity(intent)
            Hawk.deleteAll()
            finish()
        }
    }
}